front/app/components/UI/QuillEditedContent/index.tsx

```diff
@@ -25,34 +26,44 @@ interface Props {
   theme: any;
 }
 
-interface State {}
+const QuillEditedContent = ({
+  linkColor,
+  textColor,
+  mentionColor,
+  fontSize,
+  fontWeight,
+  children,
+  className,
+  theme,
+  disableTabbing,
+}: Props) => {
+  const containerRef = useRef<HTMLElement | null>(null);
 
-class QuillEditedContent extends PureComponent<Props, State> {
-  render() {
-    const {
-      linkColor,
-      textColor,
-      mentionColor,
-      fontSize,
-      fontWeight,
-      children,
-      className,
-      theme,
-    } = this.props;
+  const tabbableElements = containerRef.current?.querySelectorAll(
+    'a, iframe, button, input, select, textarea'
```

Comment:
I think we can safely exclude input, select and text area from this query selector - I don't think there's a chance they would appear in this component.